### PR TITLE
Updated the submission process for `MSRunSample` and `MSRunSequence`

### LINKED ISSUE #711: Update submission templates to ask for any information needed for `MSRunSample` and `MSRunSequence`
### Inspiration

Researchers should specify mass spec run information when submitting data.

### Description

For each peak annotation file, researchers should specify:
- 'instrument' - instrument used for this run (enum values defined by `MSRunSequence` )
- 'researcher' - name of the researcher who ran the mass spectrometer
- 'date` - the date the mass spectrometer was run
- 'notes' - any additional notes about the mass spec run (optional)

### Comment

-----

### Assumptions

- List of assumptions that the code will not explicitly address/check
- E.g. We will assume input is correct (explaining why there is no validation)

### Requirements

- [x] `1.` For each peak annotation file, researchers should specify:
   - [x] `1.1.` `instrument` - instrument used for this run
      - [x] `1.1.1.` Must be specified by enum values defined by `MSRunSequence`
   - [x] `1.2.` `researcher` - name of the researcher who ran the mass spectrometer
   - [x] `1.3.` `date` - the date the mass spectrometer was run
   - [x] `1.4.` `notes` - any additional notes about the mass spec run (optional)
   - [x] `1.5.` `polarity`

### Limitations

- A list of things this work will specifically not do
- E.g. This feature will only handle the most frequent use case X

### Affected Components

A tentative list of anticipated repository items that will be changed, labeled
with "add", "delete", or "change".  One item per line.  (Mostly, this will be a
list of files.)

- change: File path or DB table ...
- add: Environment variable or server setting
- delete: External executable or cron job

#### Interface Change description

Describe changes to usage. *E.g. GUI/command-line changes*

#### Code Change Description

Describe code changes planned for the feature. *(Pseudocode encouraged)*

#### Tests

- [ ] 1. A description of at least one test for each requirement above.
- [ ] 2. E.g. Test for req 2 that there's an exception when display value is ''
- [ ] 3. Numbers for reference & checkboxes for progress tracking

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

- updated the [submission form](https://docs.google.com/forms/d/1Y2SEQgLTd8QUjbXGCuo3989WQxMfsfh4AbJyeL3f6c8/edit)
   - Added an instrument field with radio buttons
   - Turned date field from short answer to date
   - Changed operator answer to radio buttons with an "other" option - to control name variants.  To do that, I did the following:
     ```
     ssh tracebase
     sudo -iu tracebase
     cd /var/www/tracebase
     source /usr/local/tracebase/bin/activate
     python manage.py shell
     
     In [1]: from DataRepo.models.researcher import get_researchers
     In [2]: print("\n".join(sorted(get_researchers())))
     ```
- Converted google docs links to markdown
  - Converted terms and definitions sheet using https://www.tablesgenerator.com/markdown_tables

## Affected Issues/Pull Requests

- Resolves #711

## Review Notes

I did not try generating the web content.  The links I created should probably all be checked.

## Checklist

This pull request will be merged once the following requirements are met.  The
author and/or reviewers should uncheck any unmet requirements:

- Review requirements
  - Minimum approvals: 1 
  - No changes requested
  - All blocking issues resolved by reviewers
  - Specific reviewers: @__add_username_here__
    
  - Review period: 2 days 
- Associated issue/pull request requirements:
  
  - [x] All requirements in affected issues marked "resolved" are satisfied
  - [x] All required pull requests are merged *(or none)*
- Basic requirements
  
  - [x] [All linters pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting)
  - [x] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All conflicts resolved
- Overhead requirements
  
  - [x] [New/changed method tests implemented/updated *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [x] [Updated *Unreleased* section of `changelog.md` *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/changelog.md)
  - [x] [Migrations created & committed *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)

### CODE REVIEW

File: TraceBaseDocs/TraceBaseDocs/Contributing/Report a bug or request a feature on github.md
[the remark this answers is not in the document]
Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Contributing/Report a bug or request a feature on github.md:1]: Those files were mainly exported as text from gdocs and then marked up(/down). I also tried to catch things like smart quotes and contained links to other gdocs.

File: TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md.txt
Comment (hepcat72) [inline TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md.txt:1]: C. Generated using an online markdown table generator and pasting in content from the previously linked google sheet.

File: TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+Treatment Description
+- A thorough description of an animal treatment protocol. This will be useful for searching and filtering, so use standard terms and be as complete as possible.
+- Any difference in Treatment should be indicated by a new Animal Treatment
+   - Example:  different doses of T3 in drinking water are indicated by different Animal Treatments
+   - | **Animal Treatment**            | **Treatment Description**                                                                                |
+     |---------------------------------|----------------------------------------------------------------------------------------------------------|
+     | no treatment                    | No treatment was applied to the animal. Animal was housed at room temperature with a normal light cycle. |
+     | T3 in drinking water            | T3 was provided in drinking water at 0.5 mg/L for two weeks prior to the start of infusion.              |
+     | T3 in drinking water (1.5 mg/L) | T3 was provided in drinking water at 1.5 mg/L for two weeks prior to the start of infusion.              |
```
Comment (hepcat72) [inline TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md:132]: C. Generated using a markdown table generator online.

Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md:132]: I actually converted text from an embedded image in the gdoc.

File: TraceBaseDocs/TraceBaseDocs/Contributing/Report a bug or request a feature on github.md
[the remark this answers is not in the document]
Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Contributing/Report a bug or request a feature on github.md:1]: Be sure to also review the changes to the [submission form](https://docs.google.com/forms/d/1Y2SEQgLTd8QUjbXGCuo3989WQxMfsfh4AbJyeL3f6c8/edit), which I comment on in the PR description.

Comment (hepcat72) [inline TraceBaseDocs/TraceBaseDocs/Contributing/Report a bug or request a feature on github.md:1]: C. I took advantage of the opportunity of this small update to take care of some google doc links. Multiple files were created to replace direct links to google docs.

Those files were mainly exported as text from gdocs and then marked up(/down). I also tried to catch things like smart quotes and contained links to other gdocs.

Be sure to also review the changes to the [submission form](https://docs.google.com/forms/d/1Y2SEQgLTd8QUjbXGCuo3989WQxMfsfh4AbJyeL3f6c8/edit), which I comment on in the PR description.

File: TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md
Comment (hepcat72) [inline TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md:1]: C. Generated using an online markdown table generator and pasting in content from the previously linked google sheet.

File: TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+Treatment Description
+- A thorough description of an animal treatment protocol. This will be useful for searching and filtering, so use standard terms and be as complete as possible.
+- Any difference in Treatment should be indicated by a new Animal Treatment
+  - Example:  different doses of T3 in drinking water are indicated by different Animal Treatments
+  - | **Animal Treatment**            | **Treatment Description**                                                                                |
+    |---------------------------------|----------------------------------------------------------------------------------------------------------|
+    | no treatment                    | No treatment was applied to the animal. Animal was housed at room temperature with a normal light cycle. |
+    | T3 in drinking water            | T3 was provided in drinking water at 0.5 mg/L for two weeks prior to the start of infusion.              |
+    | T3 in drinking water (1.5 mg/L) | T3 was provided in drinking water at 1.5 mg/L for two weeks prior to the start of infusion.              |
```
Comment (hepcat72) [inline TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md:106]: C. Generated using a markdown table generator online.

I actually converted text from an embedded image in the gdoc.

File: .markdown-lint.yml [hunk 3]
```diff
@@ -23,6 +23,7 @@ MD007:
   indent: 2                   # Unordered list indentation
 MD013:
   line_length: 400            # Line length 80 is far too short
+  tables: false
```
Comment (hepcat72) [inline .markdown-lint.yml:26]: C. The table content auto-wraps.  The longest line is 562.  So I learned you can set the linter to ignore line length for tables.

File: TraceBaseDocs/TraceBaseDocs/Contributing/Report a bug or request a feature on github.md
Comment (hepcat72) [inline TraceBaseDocs/TraceBaseDocs/Contributing/Report a bug or request a feature on github.md:1]: C. I took advantage of the opportunity of this small update to take care of some google doc links. Multiple files were created to replace direct links to google docs.

Those files were mainly exported as text from gdocs and then marked up(/down). I also tried to catch things like smart quotes and contained links to other gdocs.

Be sure to also review the changes to the [submission form](https://docs.google.com/forms/d/1Y2SEQgLTd8QUjbXGCuo3989WQxMfsfh4AbJyeL3f6c8/edit), which I comment on in the PR description.

File: TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md
Comment (hepcat72) [inline TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md:1]: C. Generated using an online markdown table generator and pasting in content from the previously linked google sheet.

File: TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
```
Comment (hepcat72) [inline TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md:106]: C. Generated using a markdown table generator online.

I actually converted text from an embedded image in the gdoc.

File: TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md
Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md:1]: https://www.tablesgenerator.com/markdown_tables

File: TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md (hunk 4 above)
Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md:106]: https://www.tablesgenerator.com/markdown_tables#

File: TraceBaseDocs/TraceBaseDocs/Download/Format of Downloaded Data.md
Comment (lparsons) [inline TraceBaseDocs/TraceBaseDocs/Download/Format of Downloaded Data.md:1]: B: This file seems like it duplicates some information and should be combined with `TraceBaseDocs/TraceBaseDocs/Types of Data Output/Downloaded Data Formats.md`

File: TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md
Comment (lparsons) [inline TraceBaseDocs/TraceBaseDocs/Upload/How to label sample information.md:1]: B: This seems duplicative and should be combined with `TraceBaseDocs/TraceBaseDocs/Upload/Sample Information Sheet.md`

**Review by lparsons — CHANGES_REQUESTED** [review]
Looks great @hepcat72. 

The submission form seems much nicer and easier (multiple choice questions are easy to answer). 

The cleanup of the docs is a big bonus, thanks for catching that and updating the markdown. I did push a small commit to fix some broken links I discovered when building the docs (some predated this PR, but they were simple file path fixes). There are a few files that I think should be combined (or just stick to one version). Those are small fixes though and really just some additional cleanup.

**Comment by hepcat72** [comment]
> Looks great @hepcat72.
> 
> The submission form seems much nicer and easier (multiple choice questions are easy to answer).
> 
> The cleanup of the docs is a big bonus, thanks for catching that and updating the markdown. I did push a small commit to fix some broken links I discovered when building the docs (some predated this PR, but they were simple file path fixes). There are a few files that I think should be combined (or just stick to one version). Those are small fixes though and really just some additional cleanup.

Thanks for fixing the links.  That's exactly what I was hoping would come out of the review.

I hadn't intended, in this PR, to change any content (other than to satisfy linting errors).  I just transferred the content from google docs to markdown files.  While I agree with your issues, I propose we merge this and make a separate issue to fix the content?

**Review by lparsons — APPROVED** [review]
> I hadn't intended, in this PR, to change any content (other than to satisfy linting errors). I just transferred the content from google docs to markdown files. While I agree with your issues, I propose we merge this and make a separate issue to fix the content?

OK, we can do that. It just looked to me like some Google Docs were already converted to markdown and reorganized and this PR added duplicates. It'd be nice to avoid those, but if you create issues that indicate those duplicates we can tackle them individually.

**Comment by hepcat72** [comment]
> > I hadn't intended, in this PR, to change any content (other than to satisfy linting errors). I just transferred the content from google docs to markdown files. While I agree with your issues, I propose we merge this and make a separate issue to fix the content?
> 
> OK, we can do that. It just looked to me like some Google Docs were already converted to markdown and reorganized and this PR added duplicates. It'd be nice to avoid those, but if you create issues that indicate those duplicates we can tackle them individually.

Oh, well that may be?  I had just gone through every `docs.google.com` link in the files and assumed those pages had not been converted since the direct links to google existed in the docs.  In fact, I pondered changing things, like those pages that were 2 lines long - just incorporate them into the doc that linked them... but I decided I was getting too far down a rabbit hole.

I'll look to see if I can detect entire file duplicates and make an issue.  If I can't precisely identify them, I'll create a general issue linking this PR comment.

File: TraceBaseDocs/TraceBaseDocs/Download/Format of Downloaded Data.md
Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Download/Format of Downloaded Data.md:1]: Created issue

File: TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md
[the remark this answers is not in the document]
Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Overview/TraceBase Terms and Definitions.md:1]: Created issue #801

**Comment by hepcat72** [comment]
OK, I had inferred you were suggesting that the conversion had maybe ended up duplicating content in general, but I think I misinterpreted, so I just created issues for the 3 review issues you raised.

**Pull request merged**